README_for_direct_archiver.md

```diff
@@ -0,0 +1,36 @@
+# How to test/trigger the direct archiver
+
+There is a script to trigger the direct archiver under the repository's root folder: `submit_to_archives.py`.
+It needs the `submission_UUID` as the input parameter.
+
+There is 1 mandatory configuration parameter, too.
+You need to setup an environment variable for `ARCHIVER_API_KEY`. You can get this value from AWS Secret Manager.
+It is under the `ingest/archiver/wrangler/secrets` secret storage.
+
```

Comment:
The wranglers don't have AWS console access. You might want to put here the actual AWS CLI command to get that secret.

Reply:
It would also be nice to put which entities are being archived already and which are not.

Reply:
I would add an example commandline